hack/build_lua_deps.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+# Copyright 2015 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+export BUILD_PATH=rootfs/etc/nginx/lua/vendor
+export CURR_PATH="$(dirname "$0")"
+
+get_src()
+{
+  hash="$1"
+  url="$2"
+  f=$(basename "$url")
+
+  curl -sSL "$url" -o "$f"
+  tar xzf "$f"
+  rm -rf "$f"
+}
+
+cd "$(dirname "$0")"; cd ..; cd "$BUILD_PATH"
+rm -rf resty; mkdir resty; cd resty
```

Comment:
we should not assume resty is the only dependency - I think it would make more sense to delete vendor folder completely and recreate

Reply:
This is sort of thinking ahead but on my sticky branch there is a crypto folder within vendor that has files (md5, sha, etc) that are hard coded and not imported. These could be moved to a new folder if you'd prefer that the entire vendor folder be deleted.

Reply:
I see. Maybe it would make it either to categorize the vendors into two groups? I don't know yet what's the best solution.

If we don't delete the whole vendor folder then we would have to enumerate all the things here